### DISCUSSION TITLE
Kodadot / Hiring for NFTs

### DISCUSSION THREAD

**Discussion by btcmacroecon** [kodadot/nft-gallery#2433] [General]
x

**Comment by damskyftw** [comment]
Interesting read, althought i can't understand much from it what would be the real "doing". I see you have plenty of ideas, not sure if KodaDot can host all of them, but nowadays we are built mostly by contributors. Meaning if you do some work and the work is well done, you get paid. We have stewards channel for that and also couple pages in our docs. For me personally it's quite hard to read between the lines of all the text - i feel like it's all mashed up together and i don't get what exactly are you trying to solve with those arenas for example.

Regarding the "part time pay"  - as of what i could read from what you are proposing, it's not really clear what exactly you would be doing or what are the actual steps, i'm getting lost a bit :D :)

Since you said you are more of technical person, but without coding skills, you might have a [look ](https://docs.kodadot.xyz/#contests-and-other-incentive-programs)at our incentives for non-developers and maybe start with some easy-not that wide tasks there. What do you think?

**Comment by azazime** [comment]
hey tucson longtime, tried to reach you on discord
